### DISCUSSION TITLE
root.ling这个含义是什么？

### DISCUSSION THREAD

**Discussion by ArcDevil9** [apache/iotdb#3860] [Q&A] [answered]
使用的0.11.4的release版本，实际上没没有创建root.ling，但是在执行COUNT TIMESERIES root GROUP BY LEVEL=1的时候会显示
![image](https://user-images.githubusercontent.com/48848130/131202500-cc17c920-702c-4d94-b421-e6084513380d.png)

在文档里面的示例中也没有显示root.ling这个时间序列
![image](https://user-images.githubusercontent.com/48848130/131202570-96b66b64-591f-4db3-ab00-6c04cba89d1d.png)

**Comment by HTHou** [comment, marked as answer]
试试 `show storage group`。
会不会是创建过 storage group 但是这个storage group下没有timeseries？

**Reply by ArcDevil9** [reply]
是的，确实是有这个storage group，可能是我对这个时间序列理解的不清楚，我理解的时间序列是应该通过creat timeseries命令显式的创建出来的，按这个结果的含义来看，是否可以这样理解：其实一个storage group也是一种时间序列了。即任意的一个path都可以是一个时间序列？
![image](https://user-images.githubusercontent.com/48848130/131204965-bab55aa5-918a-4cf5-ab25-04cde3aa4d71.png)

![image](https://user-images.githubusercontent.com/48848130/131205307-e02a38ac-f0b9-4a98-9f1e-e7676eb0144a.png)

**Reply by HTHou** [reply]
> 其实一个storage group也是一种时间序列了。即任意的一个path都可以是一个时间序列？

不是的，storage group 可以理解为是关系数据库里的DB，是时间序列的上级结构。

**Reply by ArcDevil9** [reply]
嗯，我之前是这样理解的。如果按照这种概念，执行count timeseries命令的时候，这个结果是不是就考虑不显示没有timeseries序列的空的storage group了？否则就这个结果必然是0，也没什么显示的意义，而且还容易让人误解。

**Reply by HTHou** [reply]
单独执行 count timeseries 不会有你说的问题，显示0 是因为 group by level = 1 的效果。因为 storage group 也是元数据树的一个层级，因此在 group by 的时候肯定也会去显示这一层级下的timeseries 个数。

**Reply by qiaojialin** [reply]
把这个元数据看做一棵树，根节点是 root，turbine 和 sgcc 是 根节点下的两个节点， count timeseries  group by level 就是查看一个层级的各个节点下有没有时间序列。

返回结果的第一列只是节点而已。
